pylhc/plot_spectrum.py

```diff
@@ -10,13 +10,12 @@
 
 required arguments:
     --files                 List with basenames of Tbt files, ie. tracking.sdds
-    --working_directory     Directory containing the Frequency files, amplitude, etc.
     --harpy_directory       Directory to write results to
```

Comment:
In the help of the parameters it says, this is the directory for the Frequency files etc.